### DISCUSSION TITLE
My Nuxt app and fastApi run fast on local but when i deploy on local network server I take time.

### DISCUSSION THREAD

**Discussion by himanshubohra13** [nuxt/nuxt#21327] [Questions]
I have nuxt app whose backend is written in FastApi everything work fine in local environment but when i deploy that on local network server of company it take alot of time to get request from fastapi server 
From loading initial page under 1 sec on local server it take 10 sec. on loading initial page on loclal network of company where i deployed it.

**Comment by TheAlexLichter** [comment]
This is very hard to debug without getting hands on the actual project or a reproduction.
The issue could reach from where the app is deployed, how API requests are done and additional layers (e.g. proxies) between API and Nuxt.

**Reply by himanshubohra13** [reply]
So here is the setup my frontend run on 192.168.0.48:3000 on local network and my  backend in on 192.168.0.48:8000 and my database is in 192.168.0.38:8956 and database is actually on aws server not on local network. Mapping of port was done there. 
If i make post request from postman to 192.168.0.48:8000 ( which is backend ) it give result in 1 sec.
same request take 7 second through nuxt.
